app/src/main/java/com/example/cryptocurency/ui/adapter/SearchCoinAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.cryptocurency.ui.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.cryptocurency.R
+import com.example.cryptocurency.data.models.Coin
+import com.example.cryptocurency.databinding.ItemCoinSearchLayoutBinding
+import com.example.cryptocurency.utils.extension.loadImageSVG
+
+class SearchCoinAdapter(private val mListener: ItemClickListener<Coin>):
+    RecyclerView.Adapter<SearchCoinAdapter.ViewHolder>()  {
+
+    private var mList = mutableListOf<Coin>()
+    private var mContext: Context? = null
```

Comment:
```suggestion
    private var mContext: Context? = null
    private var mList = mutableListOf<Coin>()
```
context nên là biến ở dòng đầu tiên ấy e!